qt/WebView.cpp

```diff
@@ -359,6 +359,39 @@ WebView* WebView::createNewDocument(QWebEngineProfile* profile, const std::strin
     return webViewInstance;
 }
 
+void WebView::loadStarterScreen()
+{
+    // for startup screen - we don't have to have a COOLWSD connection
+    _document._fakeClientFd = -1;
+    _document._appDocId = 0;
+
+    QWebChannel* channel = new QWebChannel(_webView->page());
+
+    assert(_bridge == nullptr);
+    _bridge = new Bridge(channel, _document, _mainWindow, _webView.get());
+    channel->registerObject("bridge", _bridge);
+    _webView->page()->setWebChannel(channel);
+    queryGnomeFontScalingUpdateZoom();
+
+    Poco::Path coolHtmlPath(getTopSrcDir(TOPSRCDIR));
+    coolHtmlPath.append("/browser/dist/cool.html");
+    Poco::URI urlAndQuery(coolHtmlPath);
+    urlAndQuery.setScheme("file");
+    urlAndQuery.addQueryParameter("starterMode", "true");
+
```

Comment:
On a similar note, should probably check here if the system is in dark mode same as `WebView::load`
```c++
if (portalPrefersDark())
    urlAndQuery.addQueryParameter("darkTheme", "true");
```